basket/news/backends/ctms.py

```diff
@@ -873,4 +873,4 @@ def ctms_interface():
         return None
 
 
-ctms = CTMS(ctms_interface(), not settings.SFDC_ENABLED)
+ctms = CTMS(ctms_interface(), True)
```

Comment:
Nit: Just to save the grandkids having to see what `True` refers to

```suggestion
ctms = CTMS(ctms_interface(), is_primary=True)
```